migrations/V36__Add_Plan_Document_Solution_Link.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE plan_document_solution_link (
+    id UUID PRIMARY KEY NOT NULL,
+    model_plan_id UUID NOT NULL,
+    solution_id INTEGER NOT NULL,
+    document_id UUID NOT NULL,
+
+    --META DATA
+    created_by EUA_ID NOT NULL,
+    created_dts TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    modified_by EUA_ID,
+    modified_dts TIMESTAMP WITH TIME ZONE
+);
```

Comment:
I think we should have some constraints and foreign key relationships added here, eg force a valid solution_id and document_id, as well as perhaps constraining that the combination of solution / document is unique.